osh/builtin_meta.py

```diff
@@ -128,7 +128,7 @@ def Run(self, cmd_val):
                                     c_parser,
                                     self.errfmt,
                                     cmd_flags=cmd_eval.RaiseControlFlow)
-                            except vm.ControlFlow as e:
+                            except vm.IntControlFlow as e:
```

Comment:
OK so `source` can only handle regular shell `return`, not `return ()`

In retrospect, I think that was a good reason to separate them